### PR TITLE
Run ESLint

### PR DESCRIPTION
**Pull request by gkillough**

### CODE REVIEW

File: src/main/js/field/input/TableSelectInput.js [hunk 1]
```diff
@@ -255,109 +254,140 @@ class TableSelectInput extends Component {
...
+                )
                 : cell;
 
             if (cell) {
-                return <div title={cell.toString()}> {cellContent} </div>;
+                return (
+                    <div title={cell.toString()}>
+                        {' '}
+                        {cellContent}
+                        {' '}
```
Comment (bamandel) [inline src/main/js/field/input/TableSelectInput.js:283]: These empty spaces are strange, do you know why they were inserted?

Comment (gkillough) [reply src/main/js/field/input/TableSelectInput.js:283]: Looks like they were there in the original, but ESLint wants it to be explicit?

File: src/main/js/Navigation.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ class Navigation extends Component {
         const components = this.createNavItemForDescriptors(DescriptorUtilities.DESCRIPTOR_TYPE.COMPONENT, DescriptorUtilities.CONTEXT_TYPE.GLOBAL, '/alert/components/');
 
         const nav = (
-            <Fragment>
+            <>
```
Comment (psantos1113) [inline src/main/js/Navigation.js:47]: Interesting that no tag name is needed here.

Comment (gkillough) [reply src/main/js/Navigation.js:47]: Yes. I looked into this and the "shorthand notation" is preferred. `<>` is equivalent to `<Fragment>`.

File: src/main/js/field/input/TableSelectInput.js (hunk 1 above)
Comment (psantos1113) [reply src/main/js/field/input/TableSelectInput.js:283]: The dive has space before an after the cellContent so the linter to preserve those whitespace characters added the `{' '}` code to preserve the spacing

**Pull request merged**